inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -141,6 +141,59 @@ def from_random_curves(cls,
 
         return cls(coils, num_field_periods)
 
+    @classmethod
+    def from_curves_file(cls,
+                         num_field_periods,
+                         num_coils,
+                         coil_currents,
+                         curves_file,
+                         delimiter=',',
+                         max_order=14):
+        """Create StellaratorCoils from Fourier coefficients loaded from a file.
+
+        This function loads a file containing Fourier coefficients for several 
+        coils. The file is expected to have `6*num_coils` many columns, and 
+        `order+1` many rows. The columns are in the following order,
```

Comment:
So the parameters `num_coils` and `max_order` aren't needed, right? They can be inferred from the file.